docs/pages/docs/01-cloud-deployment/gc-deployment.md

```diff
@@ -2,25 +2,38 @@
 title: Google Cloud Deployment
 keywords: cloud, deployment, google
 tags: [getting-started, deployment, cloud]
-summary: "Deploying Grakn on Google Cloud"
+summary: "Deploying Grakn on Google Cloud Platform"
 sidebar: documentation_sidebar
 permalink: /docs/cloud-deployment/gc-deployment
 folder: docs
 ---
 
-# Google Cloud Launcher
+# GCP Marketplace
 
 ## Deployment
-We shall begin with deployment of a Grakn cluster. The procedure is straight-forward and takes advantage of the
-[Cloud Launcher](https://console.cloud.google.com/launcher/details/grakn-public/grakn-kgms-premium):
+We shall begin with deployment of a Grakn cluster. The procedure is straight-forward and takes advantage of the listing in [GCP Marketplace](https://console.cloud.google.com/marketplace/details/grakn-public/grakn-kgms-premium):
 
 ![](/images/gc-solution-listing.png)
 
 To start deployment, click on the `Launch on Compute Engine` button which will take you to the configuration screen
 
 ![](/images/gc-deployment-options.png)
 
-Feel free to adjust the settings to your needs. When satisfied with the configuration press `Deploy`
+Feel free to adjust the settings to your needs.
+
+### <a name="firewall"></a> Configuring firewall settings
```

Comment:
Do we need this link, @kasper-piskorski  ?

Reply:
same as with the cluster check